### PR TITLE
Adds script for validating cocina models changes.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
Allow testing cocina models changes.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

QA

### CODE REVIEW

**Comment by justinlittman** [comment]
Note that I will be adding documentation on this to the cocina models README in separate PR.

**Comment by justinlittman** [comment]
https://github.com/sul-dlss/cocina-models/pull/421

File: bin/validate-cocina [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    advance: num_for_progress_advance(count),
+    total: count
+  )
+end
+
+def validate(clazz, sample_size, writer)
+  objs = if sample_size
+           clazz.limit(sample_size)
+         else
+           clazz.all
```
Comment (edsu) [inline bin/validate-cocina:33]: Is this an iterator?

**Review by edsu — APPROVED** [review]

File: bin/validate-cocina (hunk 1 above)
Comment (justinlittman) [reply bin/validate-cocina:33]: Yes, and it is lazy.

**Pull request merged**